### PR TITLE
Automated cherry pick of #104151: delete stale UDP conntrack entries for loadbalancer IPs

### PR DESCRIPTION
**Pull request by aojea**
Cherry pick of #104151 on release-1.20.

#104151: delete stale UDP conntrack entries for loadbalancer IPs

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note
kube-proxy: delete stale conntrack UDP entries for loadbalancer ingress IP.
```

### CODE REVIEW

**Comment by aojea** [comment]
/assign @prameshj @thockin @danwinship 
ping @kubernetes/release-managers 
/kind bug
/sig network

**Comment by aojea** [comment]
@bowei needs approval too

**Comment by bowei** [comment]
Approval need to come from the release manager for 1.20.

**Comment by aojea** [comment]
> Approval need to come from the release manager for 1.20.

that is the cherry-pick approval, that based on this https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md is different that the code owner approval

**Comment by thockin** [comment]
Thanks!

/lgtm
/approve

**Review by cpanato — APPROVED** [review]
from RelEng:
/lgtm

**Pull request merged**
